bundles/org.openhab.core.semantics/src/main/java/org/openhab/core/semantics/model/property/Angle.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
```

Comment:
This looks like an outdated license header (saying 2020 instead of 2021). Does our generator script need an update?